### DISCUSSION TITLE
About the formular in MOBI

### DISCUSSION THREAD

**Discussion by Achineseman** [Open-Systems-Pharmacology/Forum#1004] [Questions & Problems] [answered]
What is the exact path for the amount of CYP3A4 and the start amount ?
Plasma of venous blood?Plasma of the liver pericentral?Or plasma of the liver periportal?
<img width="960" alt="屏幕截图 2021-10-28 200508" src="https://user-images.githubusercontent.com/81560924/139253418-46d8d0ad-982f-4f2e-aa2f-5ea23cc7de96.png">
I just know most of the reaction is in venous blood plasma

**Comment by Yuri05** [comment, marked as answer]
`..` refers to the **parent container** (related to the location of the reaction)
s. https://docs.open-systems-pharmacology.org/working-with-mobi/mobi-documentation/model-building-components#working-with-formulas for details.

When a simulation is created from building block, reaction will be created in every container **where all educts, products and modifiers of this reaction are present. `..` is then replaced with the path of that container.**

**Reply by Achineseman** [reply]
Thank you very much. It helps me a lot!
